### ISSUE TITLE
Jar Hell

### ISSUE THREAD

**Issue by virtualdogbert** [noamt/elasticsearch-grails-plugin#177]
I keep running into a jar hell error on start-up. I'm trying to just have a bare bones setup for a poc. 

I'm running Grails 3.0.11

So in my application.groovy I have:

```groovy
def elasticsearchVersion = '2.3.3'
ext['elasticsearch.version'] = elasticsearchVersion
elasticSearch {
//    disableAutoIndex = false
//    client.mode = 'local'
//    bulkIndexOnStartup = true
//    date.formats = ["yyyy-MM-dd'T'HH:mm:ss'Z'"]
    datastoreImpl = 'hibernateDatastore'
}
```
And I found I have to add the following to avoid a path.home error:
if I add `-Des.path.home=/ptmp/progs/elasticsearch-2.3.3`

But I end up getting the error below, wch alternates between log4j and xmlpull/xpp3_min.

Any ideas on what I can do to get past this error?

```groovy
`log4j:WARN No appenders could be found for logger (org.elasticsearch.node).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
Disconnected from the target VM, address: '127.0.0.1:38786', transport: 'socket'
2016-09-28 15:41:23,544 [main] ERROR grails.boot.GrailsApp run | Application startup failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'auditListener': Cannot resolve reference to bean 'indexRequestQueue' while setting bean property 'indexRequestQueue'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'indexRequestQueue': Cannot resolve reference to bean 'elasticSearchClient' while setting bean property 'elasticSearchClient'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'elasticSearchClient': FactoryBean threw exception on object creation; nested exception is java.lang.IllegalStateException: failed to load bundle [file:/ptmp/progs/elasticsearch-2.3.3/modules/reindex/reindex-2.3.3.jar] due to jar hell
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:359) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1475) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1220) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:537) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:755) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:759) ~[spring-context-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:480) ~[spring-context-4.1.8.RELEASE.jar:4.1.8.RELEASE]
Exception in thread "main" org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'auditListener': Cannot resolve reference to bean 'indexRequestQueue' while setting bean property 'indexRequestQueue'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'indexRequestQueue': Cannot resolve reference to bean 'elasticSearchClient' while setting bean property 'elasticSearchClient'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'elasticSearchClient': FactoryBean threw exception on object creation; nested exception is java.lang.IllegalStateException: failed to load bundle [file:/ptmp/progs/elasticsearch-2.3.3/modules/reindex/reindex-2.3.3.jar] due to jar hell
	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:117) ~[spring-boot-1.2.7.RELEASE.jar:1.2.7.RELEASE]
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:359)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:689) ~[spring-boot-1.2.7.RELEASE.jar:1.2.7.RELEASE]
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1475)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:321) ~[spring-boot-1.2.7.RELEASE.jar:1.2.7.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1220)
	at grails.boot.GrailsApp.run(GrailsApp.groovy:51) [grails-core-3.0.11.jar:3.0.11]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:537)
	at grails.boot.GrailsApp.run(GrailsApp.groovy:335) [grails-core-3.0.11.jar:3.0.11]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
	at grails.boot.GrailsApp.run(GrailsApp.groovy:324) [grails-core-3.0.11.jar:3.0.11]
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303)
	at grails.boot.GrailsApp$run.call(Unknown Source) [grails-core-3.0.11.jar:3.0.11]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48) [groovy-2.4.5.jar:2.4.5]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113) [groovy-2.4.5.jar:2.4.5]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:133) [groovy-2.4.5.jar:2.4.5]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:755)
	at exaservice.Application.main(Application.groovy:18) [main/:na]
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:759)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'indexRequestQueue': Cannot resolve reference to bean 'elasticSearchClient' while setting bean property 'elasticSearchClient'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'elasticSearchClient': FactoryBean threw exception on object creation; nested exception is java.lang.IllegalStateException: failed to load bundle [file:/ptmp/progs/elasticsearch-2.3.3/modules/reindex/reindex-2.3.3.jar] due to jar hell
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:480)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:359) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:117)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:689)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1475) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:321)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1220) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at grails.boot.GrailsApp.run(GrailsApp.groovy:51)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:537) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at grails.boot.GrailsApp.run(GrailsApp.groovy:335)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at grails.boot.GrailsApp.run(GrailsApp.groovy:324)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at grails.boot.GrailsApp$run.call(Unknown Source)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:133)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at exaservice.Application.main(Application.groovy:18)
	... 23 common frames omitted
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'indexRequestQueue': Cannot resolve reference to bean 'elasticSearchClient' while setting bean property 'elasticSearchClient'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'elasticSearchClient': FactoryBean threw exception on object creation; nested exception is java.lang.IllegalStateException: failed to load bundle [file:/ptmp/progs/elasticsearch-2.3.3/modules/reindex/reindex-2.3.3.jar] due to jar hell
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'elasticSearchClient': FactoryBean threw exception on object creation; nested exception is java.lang.IllegalStateException: failed to load bundle [file:/ptmp/progs/elasticsearch-2.3.3/modules/reindex/reindex-2.3.3.jar] due to jar hell
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:359)
	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:175) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.getObjectFromFactoryBean(FactoryBeanRegistrySupport.java:103) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1475)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getObjectForBeanInstance(AbstractBeanFactory.java:1523) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1220)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:314) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:537)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351) ~[spring-beans-4.1.8.RELEASE.	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
jar:4.1.8.RELEASE]
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
	... 33 common frames omitted
	... 23 more
Caused by: java.lang.IllegalStateException: failed to load bundle [file:/ptmp/progs/elasticsearch-2.3.3/modules/reindex/reindex-2.3.3.jar] due to jar hell
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'elasticSearchClient': FactoryBean threw exception on object creation; nested exception is java.lang.IllegalStateException: failed to load bundle [file:/ptmp/progs/elasticsearch-2.3.3/modules/reindex/reindex-2.3.3.jar] due to jar hell
	at org.elasticsearch.plugins.PluginsService.loadBundles(PluginsService.java:421) ~[elasticsearch-2.3.3.jar:2.3.3]
	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:175)
	at org.elasticsearch.plugins.PluginsService.<init>(PluginsService.java:115) ~[elasticsearch-2.3.3.jar:2.3.3]
	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.getObjectFromFactoryBean(FactoryBeanRegistrySupport.java:103)
	at org.elasticsearch.node.Node.<init>(Node.java:158) ~[elasticsearch-2.3.3.jar:2.3.3]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getObjectForBeanInstance(AbstractBeanFactory.java:1523)
	at grails.plugins.elasticsearch.ClientNodeFactoryBean$PluginEnabledNode.<init>(ClientNodeFactoryBean.groovy:241) ~[elasticsearch-1.2.0.jar:na]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:314)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[na:1.8.0_65]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[na:1.8.0_65]
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[na:1.8.0_65]
	... 33 more
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[na:1.8.0_65]
Caused by: java.lang.IllegalStateException: failed to load bundle [file:/ptmp/progs/elasticsearch-2.3.3/modules/reindex/reindex-2.3.3.jar] due to jar hell
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:80) ~[groovy-2.4.5.jar:2.4.5]
	at org.elasticsearch.plugins.PluginsService.loadBundles(PluginsService.java:421)
	at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:74) ~[groovy-2.4.5.jar:2.4.5]
	at org.elasticsearch.plugins.PluginsService.<init>(PluginsService.java:115)
	at org.codehaus.groovy.runtime.callsite.ConstructorSite.callConstructor(ConstructorSite.java:45) ~[groovy-2.4.5.jar:2.4.5]
	at org.elasticsearch.node.Node.<init>(Node.java:158)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:60) [groovy-2.4.5.jar:2.4.5]
	at grails.plugins.elasticsearch.ClientNodeFactoryBean$PluginEnabledNode.<init>(ClientNodeFactoryBean.groovy:241)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:235) [groovy-2.4.5.jar:2.4.5]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:255) [groovy-2.4.5.jar:2.4.5]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at grails.plugins.elasticsearch.ClientNodeFactoryBean.getObject(ClientNodeFactoryBean.groovy:203) ~[elasticsearch-1.2.0.jar:na]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:168) ~[spring-beans-4.1.8.RELEASE.jar:4.1.8.RELEASE]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	... 38 common frames omitted
	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:80)
Caused by: java.lang.IllegalStateException: jar hell!
	at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:74)
class: org.apache.juli.logging.Log
	at org.codehaus.groovy.runtime.callsite.ConstructorSite.callConstructor(ConstructorSite.java:45)
jar1: /ptmp/.gradle/caches/modules-2/files-2.1/org.apache.tomcat/tomcat-juli/7.0.55/7b0d3759db33865f882e77bdddb87c3cb28cecc3/tomcat-juli-7.0.55.jar
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:60)
jar2: /ptmp/.gradle/caches/modules-2/files-2.1/org.apache.tomcat.embed/tomcat-embed-logging-log4j/7.0.55/1e12baf15e15e713b3dc4353e51b0d7d18e010b8/tomcat-embed-logging-log4j-7.0.55.jar
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:235)
	at org.elasticsearch.bootstrap.JarHell.checkClass(JarHell.java:282) ~[elasticsearch-2.3.3.jar:2.3.3]
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:255)
	at org.elasticsearch.bootstrap.JarHell.checkJarHell(JarHell.java:186) ~[elasticsearch-2.3.3.jar:2.3.3]
	at grails.plugins.elasticsearch.ClientNodeFactoryBean.getObject(ClientNodeFactoryBean.groovy:203)
	at org.elasticsearch.plugins.PluginsService.loadBundles(PluginsService.java:419) ~[elasticsearch-2.3.3.jar:2.3.3]
	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:168)
	... 53 common frames omitted
	... 38 more
Caused by: java.lang.IllegalStateException: jar hell!
class: org.apache.juli.logging.Log
jar1: /ptmp/.gradle/caches/modules-2/files-2.1/org.apache.tomcat/tomcat-juli/7.0.55/7b0d3759db33865f882e77bdddb87c3cb28cecc3/tomcat-juli-7.0.55.jar
jar2: /ptmp/.gradle/caches/modules-2/files-2.1/org.apache.tomcat.embed/tomcat-embed-logging-log4j/7.0.55/1e12baf15e15e713b3dc4353e51b0d7d18e010b8/tomcat-embed-logging-log4j-7.0.55.jar
	at org.elasticsearch.bootstrap.JarHell.checkClass(JarHell.java:282)
	at org.elasticsearch.bootstrap.JarHell.checkJarHell(JarHell.java:186)
	at org.elasticsearch.plugins.PluginsService.loadBundles(PluginsService.java:419)
	... 53 more

Process finished with exit code 1
```

**Comment by puneetbehl** [comment]
Could you please upload a sample application replicating the problem?

**Comment by virtualdogbert** [comment]
Ok I created a sample app mostly just copying the build.gradle:
https://github.com/virtualdogbert/brokensearch

Although I get a slightly different jar hell error it complains about:
compile 'javax.websocket:javax.websocket-api:1.1'

and when I comment that out, I get back to it complaining about the logging libraries. However I have seen it also complain about xmlpush and some other xml jar, in my main app.

**Comment by virtualdogbert** [comment]
After looking back at the documentation there doesn't seem to be a version of the plug-in that's supported for 3.0.x, could this be the source of my problems, and frustration? I got further by trying the steps bellow, but it still doesn't really work.

I changed my build.gradle to have:
`compile ('org.grails.plugins:elasticsearch:1.2.0'){
        exclude module: 'gson'
    }
    compile ("org.elasticsearch:elasticsearch:2.3.3"){
        exclude module: 'gson'
    }`

and my application.groovy to have:
`def elasticsearchVersion = '2.3.3'
ext['elasticsearch.version'] = elasticsearchVersion
elasticSearch {
//    disableAutoIndex = false
    client.mode = 'local'
//    bulkIndexOnStartup = true
//    date.formats = ["yyyy-MM-dd'T'HH:mm:ss'Z'"]
    datastoreImpl = 'hibernateDatastore'
    index.numberOfReplicas =0
}`

and I removed:
-Des.path.home=/ptmp/progs/elasticsearch-2.3.3

While I seem to get past compilation, but if I try MyDomain.search('anything') (I did add a searchable mapping for the domain), I get a missing method exception. If I try the elasticSearchService, I get a no class def error for:
java.lang.NoClassDefFoundError: com/vividsolutions/jts/geom/Coordinate
